docs/plans/2026-01-20-runtime-refactor-implementation.md

```
@@ -0,0 +1,1493 @@
+**Files:**
+- Create: `packages/runtime/package.json`
+- Create: `packages/runtime/tsconfig.json`
+- Create: `packages/runtime/tsconfig.node.json`
+- Create: `packages/runtime/vite.config.ts`
+- Create: `packages/runtime/index.html`
+- Create: `packages/runtime/src/main.ts`
+- Create: `packages/runtime/src/client/index.ts`
+- Create: `packages/runtime/src/client/ws.ts`
+- Create: `packages/runtime/src/client/overlay.ts`
+
+**Step 1: 创建 package.json**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)